pages/common/finger.md

```diff
@@ -0,0 +1,23 @@
+# finger
+
+> User information lookup program.
+
+- Display information about currently logged in users:
+
+`finger`
+
+- Display information about a specific user:
+
+`finger {{username}}`
+
+- Display the user's login name, real name, terminal name and write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number:
```

Comment:
Remove - `write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number` and just put `other information`.

Reply:
Done.